package.json

```diff
@@ -16,7 +18,7 @@
   "main": "dist/rollup-plugin-npm.cjs.js",
   "jsnext:main": "dist/rollup-plugin-npm.es6.js",
   "scripts": {
-    "test": "mocha",
+    "test": "mocha --compilers js:babel-register",
```

Comment:
Because I keep using things like `const` and arrow functions and destructuring out of habit, and it works because I'm on Node 5. But then Travis tries to run the tests in 0.12 and I lose 5 minutes removing the ES6 stuff, committing the fixes, then re-running the tests. Figured this was easier!